vagrant/Vagrantfile

```diff
@@ -0,0 +1,16 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+# you're doing.
```

Comment:
what does this comment do?

Reply:
The first 2 line are generated when creating a Vagrant file using `vagrant up`. These lines tell different editing tools that they should use ruby highlighting if it is available.

The third line is a missed deletion of another comment.